tests/test_examples.py

```diff
@@ -0,0 +1,102 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import pytest
+from common import get_hif2a_ligands_as_sdf_file, temporary_working_dir
+from rdkit import Chem
+
+from timemachine.fe.free_energy import RABFEResult
+
+# All examples are to be tested nightly
+pytestmark = [pytest.mark.nightly]
+
+
+EXAMPLES_DIR = Path(__file__).parent.parent / "examples"
+
+
+def run_example(
+    example_name: str, cli_args: List[str], env: Optional[Dict[str, str]] = None, cwd: Optional[str] = None
+) -> subprocess.CompletedProcess:
+    """
+    Runs an example script
+
+    Parameters
+    ----------
+
+    example_name: Name of the example
+            The name of a file within the examples/ directory
+
+    cli_args: List of command line arguments to pass
+
+    env: Dictionary to override environment variables
+
+    cwd: Directory to run in, defaults in current directory
+
+    Returns
+    -------
+
+    Returns the completed subprocess
+    """
+    example_path = EXAMPLES_DIR / example_name
+    assert example_path.is_file(), f"No such example {example_path}"
+    subprocess_env = os.environ.copy()
+    if env is not None:
+        subprocess_env.update(env)
+    subprocess_args = [sys.executable, str(example_path), *cli_args]
+    proc = subprocess.run(
+        subprocess_args,
+        env=subprocess_env,
+        check=True,
+        cwd=cwd,
+    )
+    return proc
+
+
+def test_relative_binding():
+    """
+    Test validate_relative_binding.py to ensure that it functions correctly
+    """
+    protein_path = str(Path(__file__).resolve().parent / "data" / "hif2a_nowater_min.pdb")
+    temp_ligands = get_hif2a_ligands_as_sdf_file(1)
+
+    # Any fewer windows and the lambda schedule validation will fail
+    windows = str(5)
```

Comment:
Surprising...

Should we modify the `> 0` to ` >= 0`? https://github.com/proteneer/timemachine/blob/56d9933d208b07bfa0524376086f27a26a468f82/timemachine/fe/lambda_schedule.py#L32-L37

Reply:
If I remember correctly it was the linear interpolation that allows the endpoints to not be 0.0 and 1.0

Reply:
>If I remember correctly it was the linear interpolation that allows the endpoints to not be 0.0 and 1.0

Should I move this to a new issue?

Locally it looks like this is a problem with the methods that use `linspace` and `concatenate`:
* construct_absolute_lambda_schedule_solvent
* construct_absolute_lambda_schedule_complex
* construct_relative_lambda_schedule

but not with the method that uses `interp`
* construct_pre_optimized_absolute_lambda_schedule_solvent

Reply:
Ah that is the same for me, was wrong about which schedule it was.

A new issue seems fine, though I don't know we have much utility for tiny numbers of windows.

Reply:
Will open a new issue in a second!

Reply:
Moved to https://github.com/proteneer/timemachine/issues/714